game/script-topics.rpy

```diff
@@ -11550,3 +11551,409 @@ label talk_enable_no_topics_reminder:
+
+        else:
+            n 3tlrbo "Work experience,{w=0.5}{nw}"
+            extend 3tnmbo " huh?"
+            n 4ccsflsbl "...Wait.{w=0.75}{nw}"
+            extend 4nsqflsbl " Don't tell me.{w=0.75}{nw}"
+            extend 2csqem " You mean like some kind of placement while I was in school?{w=0.75}{nw}"
+            extend 2tsqfl " Or like an internship?"
+            n 1cnmsl "{i}That{/i} kind of work experience?"
+            n 2fcsslesi "..."
+            n 2ccsfl "Heh.{w=0.75}{nw}"
+            extend 2cllfl " Yeah.{w=0.75}{nw}"
+            extend 4cslfr " The whole process was an {i}experience{/i} alright."
+            n 7nllaj "So.{w=0.75}{nw}"
+            extend 7ullfl " I don't know how it works for you,{w=0.2} [player].{w=0.75}{nw}"
+            extend 3tnmsl " But at my school at least?{w=0.75}{nw}"
+            extend 3clrem " We didn't even get to properly {i}choose{/i} our placements."
+
+        n 1fnmem "Seriously -{w=0.5}{nw}" 
+        extend 4fllgs " it was a total crapshoot!{w=0.75}{nw}"
+        extend 2fslan " Nothing but dumb luck!"
+
+        if already_discussed_work_experience:
+            n 2ccsslsbr "And no,{w=0.2} like I said -{w=0.5}{nw}"
+
+        else:
+            n 2ccsemsbr "And no,{w=0.2} [player],{w=0.2} before you ask -{w=0.5}{nw}"
+
+        extend 1csqemsbr " we didn't get a choice on if we {i}wanted{/i} to do it either."
+        n 4clraj "We all had this online form we had to fill out in the computer room,{w=0.5}{nw}"
+        extend 4fsqfl " but it wasn't like we could just type in a place we already had in mind.\n{w=0.75}{nw}"
+        extend 2fupem "Of {i}course{/i} not."
+        n 2fcsgs "Nope!{w=0.75}{nw}"
+        extend 2fllem " It was just a bunch of pre-filled options we had to pick between.{w=0.75}{nw}"
+        extend 4fslan " They weren't even good ones!"
+        n 2flrem "There were a couple of office places,{w=0.2} sure."
+        n 2fnman "But most of them were just working in some random store!{w=0.75}{nw}"
+        extend 4fcswr " The whole point of those is that you don't even {i}need{/i} experience to do them!"
+        n 2fsrem "...And we had to pick {i}three{/i} of them too.{w=0.5} {i}In order of preference{/i}."
+        n 1fcssl "Heh.{w=0.75}{nw}" 
+        extend 4fsqfl " The worst part,{w=0.2} [player]?{w=0.75}{nw}"
+
+        if already_discussed_work_experience:
+            extend 4fslsl " It {i}still{/i} bugs me talking about it a second time."
+
+        else:
+            extend 4fllfl " You're not even gonna {i}believe{/i} this."
+
+        n 4fnmgs "We had to write out a complete cover letter for all of them too!{w=0.75}{nw}"
+        extend 2fsran " There was a word count and everything!"
+        n 2ccsemesi "..."
+        n 2fcsbosbr "I get it.{w=0.75}{nw}"
+        extend 7cllflsbl " It was supposed to give us practice on how to apply for stuff in the future."
+        n 1fcsaw "But come{w=0.5}{nw}"
+        extend 3fcsgs " {i}on{/i}!"
```

Comment:
Tbh maybe it's because I'm already reviewing for a while, but it seems like Natsuki often uses this phrase. 
Of course it can just be the way she is talking, but it feels repetitive after a while and we don't want the player to have a unique experience with every topic

Reply:
I can tweak it, we use it a lot as it shows off exasperation well in a dynamic way, but I agree